platforms/ucl_myriad/nextflow.config

```diff
@@ -0,0 +1,15 @@
+
+executor {
+    name = 'sge'
+    queueSize = 10
+}
+
+process {    
+    executor = 'sge'
+    beforeScript = "source ${launchDir}/platforms/ucl_myriad/pre.sh"
+    cpus = 1
+    memory = '4 GB'
+    time = '10min'
+    clusterOptions ='-pe smp 4 -l mem=3G,h_vmem=1G,h_rt=1:0:0 -N cath_job'
```

Comment:
Probably best to make sure cpus and pe smp values match? nextflow will autogenerate a job name so -N option maybe not required? with SGE I think it doesn't make sense to put mem larger than h_vmem as virtual memory (ram + swap I think) is always >= true memory (ram only)

Reply:
For the cluster options, can I leave these in for testing and tuning? These are the tuned options that the nf-core config ues so may not make sense but can we start with them and change after?

Reply:
Agreed (as we said when we spoke last week)